src/main/java/mate.academy/lib/Component.java

```diff
@@ -1,5 +1,9 @@
 package mate.academy.lib;
 
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+
+@Retention(RetentionPolicy.RUNTIME)
```

Comment:
![image](https://github.com/mate-academy/jv-dependency-injection/assets/49767049/8ef00242-0808-43c9-9563-f70316e4125c)

Reply:
I'm sorry, I'll fix it. I'll be more careful